danielgrj/chip-8-emulator
# Add jest config

**Description**
This PR adds missing the jest config

**To tests**
- Run the script `yarn test`. It should start the watch mode.
- Try making a commit, the test suit should run in coverage mode.

**Dependencies added**
- is-ci-cli
- jest
- jest-emotion
- jest-runner-eslint
- jest-watch-select-projects
